modules/remotebackend/httpconnector.cc

```diff
@@ -418,8 +418,9 @@ int HTTPConnector::recv_message(Json& output) {
 
     arl.finalize();
 
-    if (resp.status < 200 || resp.status >= 400) {
+    if ((resp.status < 200 || resp.status >= 400) && resp.status != 404) {
       // bad. 
+      g_log<<Logger::Warning<<"Received unacceptable HTTP status code "<<std::to_string(resp.status)<<" from HTTP endpoint"<<endl;
```

Comment:
should this be DBException instead?

Reply:
hmm, I typed that in, but I _think_ blowing up the connection on a 404 is a bit much.....

Reply:
uhm, the if condition precludes 404, so it's not blowing up the connection.. I am bit puzzled how 404's are treated now.

Reply:
should there be unit/integration test for that?

Reply:
> uhm, the if condition precludes 404, so it's not blowing up the connection.. I am bit puzzled how 404's are treated now.

ah yes :)

> should there be unit/integration test for that?

It is implicitly tested, as I got exceptions on 404s before :)

Reply:
This looks resolved to me, except for the exception type, right?